uninstall

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+# POSIX
+
+dest_path="/usr/local/bin"
+verbose=0
+
+# Print usage to stdout.
+# Arguments:
+#   None
+# Outputs:
+#   print usage with examples.
+show_help() {
+	cat <<EOF
+uninstall -p /usr/local/bin
+
+Options:
+	--path/-p:		path where the files are located (default /usr/local/bin)
+	--help/-h:		show this message
+	-v:			show more verbosity
+EOF
+}
+
+# Parse arguments
+while :; do
+	case $1 in
+	-h | --help)
+		# Call a "show_help" function to display a synopsis, then exit.
+		show_help
+		exit
+		;;
+	-v | --verbose)
+		shift
+		verbose=1
+		;;
+	-p | --path)
+		if [ -n "$2" ]; then
+			dest_path="$2"
+			shift
+			shift
+		fi
+		;;
+	*) # Default case: If no more options then break out of the loop.
+		break ;;
+	esac
+done
+
+set -o errexit
+set -o nounset
+# set verbosity
+if [ "${verbose}" -ne 0 ]; then
+	set -o xtrace
+fi
+
+# Ensure the foundamental variables are set and not empty, we will not proceed if
+# they are not all set.
+[ ! -d "${dest_path}" ] && printf >&2 "Path %s does not exist.\n" "${dest_path} " && exit 1
+[ ! -w "${dest_path}" ] && printf >&2 "Cannot write into %s, permission denied.\n" "${dest_path}" && exit 1
+
+# uninstall
+if [ -f "${dest_path}/distrobox-enter" ]; then
```

Comment:
Instead of this, we can simply have the list of files declared explicitly (enter, create etc etc)
And loop over that, so that if (for example) an user removes create file, but leaves the others, the uninstaller will work anyway

Reply:
I quite like this approach to be honest